### ISSUE TITLE
Tarball builds get big warning from git init about use of "master" as the primary branch

### ISSUE THREAD

**Issue by dougburns** [ZOSOpenTools/meta#205]
e.g.
```
Extract tarball findutils-4.9.0.tar.xz into findutils-4.9.0
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
```

I think we should be following best practice when creating local git repositories and explicitly request "main".

**Comment by MikeFultonDev** [comment]
Is this something we should:
- change in our `gitport` so that the generated .env sets this
- recommend it to developers via our git faq?
- check it in zopen-build before issuing a `git init` ?

My thoughts are we:
- put it in the FAQ
- check it in zopen-build and fail if it's not set to main, telling people how to fix it

What do others think?

**Comment by MikeFultonDev** [comment]
Ah. you already implemented it. perfect.

**Comment by MikeFultonDev** [comment]
fixed in PR #206
